### PR TITLE
CFY 1963 openstack credentials not uploaded to manager correctly

### PR DESCRIPTION
**Pull request by ran-z**

### CODE REVIEW

File: openstack-docker/tests/__init__.py [hunk 1]
```diff
@@ -0,0 +1 @@
+__author__ = 'ran'
```
Comment (idanmo) [inline openstack-docker/tests/__init__.py]: author...

File: softlayer/tests/__init__.py [hunk 2]
```diff
@@ -0,0 +1 @@
+__author__ = 'ran'
```
Comment (idanmo) [inline softlayer/tests/__init__.py]: ....

File: test_utils/__init__.py [hunk 3]
```diff
@@ -0,0 +1 @@
+__author__ = 'ran'
```
Comment (idanmo) [inline test_utils/__init__.py]: ....

File: cloudstack/tests/__init__.py [hunk 4]
```diff
@@ -0,0 +1 @@
+__author__ = 'ran'
```
Comment (idanmo) [inline cloudstack/tests/__init__.py]: remove

Comment (idanmo) [inline cloudstack/tests/__init__.py]: remove

File: openstack-docker/tests/test_openstack_docker_blueprint.py [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+            'auth_url': 'file-auth-url'
+        }
+        conf_file_path = tempfile.mkstemp()[1]
+        os.environ[Config.OPENSTACK_CONFIG_PATH_ENV_VAR] = conf_file_path
+        with open(conf_file_path, 'w') as f:
+            json.dump(file_config, f)
+
+        os.environ['OS_USERNAME'] = 'envar-username'
+        os.environ['OS_PASSWORD'] = 'envar-password'
+        os.environ['OS_TENANT_NAME'] = 'envar-tenant-name'
```
Comment (idanmo) [inline openstack-docker/tests/test_openstack_docker_blueprint.py]: Don't you want to test that all relevant OS env variables are passed to configuration?

File: openstack-nova-net/tests/test_openstack_nova_net_blueprint.py [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+            'auth_url': 'file-auth-url'
+        }
+        conf_file_path = tempfile.mkstemp()[1]
+        os.environ[Config.OPENSTACK_CONFIG_PATH_ENV_VAR] = conf_file_path
+        with open(conf_file_path, 'w') as f:
+            json.dump(file_config, f)
+
+        os.environ['OS_USERNAME'] = 'envar-username'
+        os.environ['OS_PASSWORD'] = 'envar-password'
+        os.environ['OS_TENANT_NAME'] = 'envar-tenant-name'
```
Comment (idanmo) [inline openstack-nova-net/tests/test_openstack_nova_net_blueprint.py]: Same comment as openstack docker..

File: openstack-docker/tests/test_openstack_docker_blueprint.py (hunk 5 above)
Comment (ran-z) [inline openstack-docker/tests/test_openstack_docker_blueprint.py]: this is not a test for the plugin itself - the test is for the correct copying of the plugin's output after merging with the credentials inputs. thats why im checking for the correct overrides and merging behavior, but only for a sample.

File: openstack-nova-net/tests/test_openstack_nova_net_blueprint.py (hunk 6 above)
Comment (ran-z) [inline openstack-nova-net/tests/test_openstack_nova_net_blueprint.py]: same reply :p

**Pull request merged**